### DISCUSSION TITLE
How to set FOV of Camera

### DISCUSSION THREAD

**Discussion by Orics** [mozilla/hubs#5939] [Spoke Questions] [answered]
Hi. 
I'm working on a project with hubs.
But FOV (field of view) so large. So, can I change FOV of camera? How to change it?
![image](https://user-images.githubusercontent.com/32889534/218033405-023469a9-4c36-4e82-9b81-0d67e6bfe153.png)

**Comment by rawnsley** [comment, marked as answer]
There’s an FoV URL parameter: https://hubs.mozilla.com/docs/hubs-query-string-parameters.html

**Comment by Orics** [comment]
Thank you
